contracts/PartyBid.sol

```diff
@@ -59,10 +59,17 @@ contract PartyBid is ReentrancyGuardUpgradeable, ERC721HolderUpgradeable {
 
     // ============ Internal Constants ============
 
+    // PartyBid version 1.2
```

Comment:
i guess we should just call this 2 now?

Reply:
cool - renamed in code (https://github.com/PartyDAO/partybid/pull/56/commits/9e2ff0983714d7f5dc591b2bc15577ea904a1d0b) and in PR / commit messages